File: src/main/java/fi/metatavu/metaform/server/persistence/model/ListReplyField.java
package fi.metatavu.metaform.server.persistence.model;

import javax.persistence.Cacheable;
import javax.persistence.Entity;

import org.hibernate.annotations.Cache;
import org.hibernate.annotations.CacheConcurrencyStrategy;

/**
 * JPA entity representing list field in reply
 * 
 * @author Antti Leppä
 */
@Entity
@Cacheable(true)
@Cache(usage = CacheConcurrencyStrategy.TRANSACTIONAL)
public class ListReplyField extends ReplyField {

}

File: src/main/kotlin/fi/metatavu/metaform/server/persistence/model/ListReplyField.kt
package fi.metatavu.metaform.server.persistence.model

import jakarta.persistence.Cacheable
import jakarta.persistence.Entity

/**
 * JPA entity representing list field in reply
 *
 * @author Antti Leppä
 */
@Entity
@Cacheable(true)
class ListReplyField : ReplyField()